Makefile

```
@@ -74,15 +74,32 @@ test-watch: ## Run tests in watch mode (requires pytest-watch)
 ##@ Code Quality
 
 .PHONY: format
-format: ## Format code with black, isort, and markdownlint
+format: ## Format code with black, isort, markdownlint, and terraform fmt
```

checkmake (0.2.2)

[warning] Target body for "format" exceeds allowed length of 5 (12).

(maxbodylength)